lib/datalink.coffee

```diff
@@ -145,6 +148,10 @@ class Datalink
 
   # Returns the deploy job status.
   jobStatus: (job, cb) =>
+    if not job?
+      lastJob = project.lastJobId
+      if not lastJob? then return cb new Error 'No previous job stored. Please provide a job ID.'
+      job = lastJob
```

Comment:
this can probably be simplified as
```
if not job?
  job = project.lastJobId
  if not job? then return cb new Error 'No previous job stored. Please provide a job ID.'
```